geyser-plugin-manager/Cargo.toml

```diff
@@ -13,6 +13,8 @@ documentation = "https://docs.rs/solana-geyser-plugin-manager"
 bs58 = "0.4.0"
 crossbeam-channel = "0.5"
 json5 = "0.4.1"
+jsonrpc-core = "18.0.0"
+jsonrpc-server-utils = "18.0.0"
```

Comment:
It would be best not to add jsonrpc dependency here. You can add a custom error variant for the Result.
Is there a reason we can't use tokio directly for the channel?

Reply:
Tokio is not a direct dependency. I could add it.

Reply:
I had actually planned to add another error variant, but that was before we said it was okay to add a dependency for the Request enum. I could add it now.